core/src/main/res/values/strings.xml

```diff
@@ -407,102 +407,102 @@
+    <string name="pref_compact_notification_buttons_dialog_error">Select less than %1$d items.</string>
     <string name="pref_lockscreen_background_title">Set Lockscreen Background</string>
-    <string name="pref_lockscreen_background_sum">Set the lockscreen background to the current episode\'s image. As a side effect, this will also show the image in third party apps.</string>
+    <string name="pref_lockscreen_background_sum">Use current episode\'s image in external players and as lockscreen image.</string>
     <string name="pref_enqueue_location_title">Enqueue Location</string>
     <string name="pref_enqueue_location_sum">Add episodes to: %1$s</string>
     <string name="enqueue_location_back">Back</string>
     <string name="enqueue_location_front">Front</string>
     <string name="enqueue_location_after_current">After current episode</string>
-    <string name="pref_smart_mark_as_played_disabled">Disabled</string>
-    <string name="documentation_support">Documentation &amp; Support</string>
-    <string name="visit_user_forum">User forum</string>
-    <string name="bug_report_title">Report bug</string>
+    <string name="pref_smart_mark_as_played_disabled">Off</string>
+    <string name="documentation_support">Documentation and Support</string>
+    <string name="visit_user_forum">Forum</string>
+    <string name="bug_report_title">Report Bug</string>
     <string name="open_bug_tracker">Open bug tracker</string>
     <string name="copy_to_clipboard">Copy to clipboard</string>
     <string name="copied_to_clipboard">Copied to clipboard</string>
     <string name="experimental_pref">Experimental</string>
-    <string name="pref_media_player_message">Select which media player to use to play files</string>
+    <string name="pref_media_player_message">Select a media player to play files with</string>
     <string name="pref_current_value">Current value: %1$s</string>
     <string name="pref_proxy_title">Proxy</string>
     <string name="pref_proxy_sum">Set a network proxy</string>
     <string name="pref_no_browser_found">No web browser found.</string>
-    <string name="pref_cast_title">Chromecast support</string>
-    <string name="pref_cast_message_play_flavor">Enable support for remote media playback on Cast devices (such as Chromecast, Audio Speakers or Android TV)</string>
-    <string name="pref_cast_message_free_flavor" tools:ignore="UnusedResources">Chromecast requires third party proprietary libraries that are disabled in this version of AntennaPod</string>
+    <string name="pref_cast_title">Cast Device Support</string>
+    <string name="pref_cast_message_play_flavor">Playback on external Cast devices (like Chromecast, Audio Speakers or Android TV)</string>
+    <string name="pref_cast_message_free_flavor" tools:ignore="UnusedResources">Chromecast requires proprietary third-party libraries not found in this libre version of AntennaPod.</string>
```

Comment:
'not found' sounds like it should be there but isn't. That's not the case (it's a conscious act). Also, 'this libre version' suggests that the other version(s) are not (while they are also released under a libre/open source license):
```suggestion
    <string name="pref_cast_message_free_flavor" tools:ignore="UnusedResources">Chromecast requires proprietary third-party libraries not available in this version of AntennaPod.</string>
```